sbin/build.sh

```diff
@@ -279,18 +279,14 @@ configureVersionStringParameter() {
     if isGnuCompatDate; then
```

Comment:
Since `--utc` is a synonym for `-u` on GNU date, presumably this if is redundant now?

Reply:
Good spot :-)